content/doc/developer/plugin-governance/adopt-a-plugin.adoc

```diff
@@ -20,23 +20,44 @@ This page addresses cases when there is no active maintainer of a plugin:
 
 == Requesting commit access
 
-Email the http://jenkins-ci.org/content/mailing-lists[Jenkins Developers mailing list] and request to be made a maintainer 
-(which usually means commit access to the plugin repository and being made default assignee for newly reported issues in JIRA).
-Please provide the following things in the request:
+=== Plugins marked for adoption
+
+The procedure is two-fold: 
+
+==== announce your adoption intention by mail 
+
+Send the notification email to the http://jenkins-ci.org/content/mailing-lists[Jenkins Developers mailing list].
```

Comment:
Better to link directly to the mailing list
```suggestion
Send the notification email to the https://groups.google.com/g/jenkinsci-dev[Jenkins Developers mailing list].
```

Reply:
If you prefer to remain with the link to the lists page, please switch the link to `link:/mailing-lists/` so that it will work for local site development and will use HTTPS immediately instead of redirecting through HTTP to HTTPS